rebasehelper/application.py

```diff
@@ -708,7 +717,7 @@ def run(self):
                     Application.check_build_requires(self.spec_file)
                 # Build packages
                 try:
-                    build = self.build_packages()
+                    build = self.build_packages
```

Comment:
This is for sure a typo. the build_packages function is not called, but instead stored in the build variable. Please fix it.